references/attributes_for_faceting.md

```diff
@@ -0,0 +1,95 @@
+# Attributes for Faceting
+
+_Child route of the [settings route](/references/settings.md)._
+
+The attributes that can be used as [facets for faceted search](/guides/advanced_guides/faceted_search.md).
+
+Attributes for faceting can also be updated directly through the [global settings route](/references/settings.md#update-settings) at the same time than the other settings.
```

Comment:
```suggestion
Attributes for faceting can also be updated directly through the [global settings route](/references/settings.md#update-settings) along with the other settings.
```

Reply:
Ouf this is way better. I will change it everywhere